docs/cqrs/pipeline/index.md

```diff
@@ -0,0 +1,138 @@
+# Pipeline
+
+The LeanCode CoreLibrary utilizes ASP.NET middlewares to create customized pipelines for handling requests and responses. This section intends to showcase the setup of a basic pipeline and explore its inner workings.
```

Comment:
We're in context of the CQRS pipelines, not arbitrary pipelines. I would not use "requests" and "responses" here.